openreview/conference/templates/programchairWebfield.js

```diff
@@ -717,13 +718,23 @@ var buildSeniorAreaChairGroupMaps = function(noteNumbers, seniorAreaChairGroups)
   };
 };
 
-var getOfficialReviews = function(notes) {
+var getOfficialReviews = function (notes, anonReviewerGroups, submissionNumber) {
   var ratingExp = /^(\d+): .*/;
 
   var reviewByAnonId = {};
 
   _.forEach(notes, function(n) {
-    var anonId = getNumberfromGroup(n.signatures[0], ANON_REVIEWER_NAME);
+    var anonId;
+    if (IS_NON_ANONYMOUS_VENUE) {
```

Comment:
instead of having a variable I would check if the signature is a tilde id:

```
anonId = getNumberfromGroup(n.signatures[0].startsWith('~') ? submissionAnonReviewerGroups[reviewerProfileId]) : n.signatures[0], ANON_REVIEWER_NAME);
```

Reply:
the variable is also used in other parts of the webfield.
the approach of having a local var and update it here when it first see a signature starts with ~ 
may fail if all existing reviews are signed using for example the pc group